source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2140,35 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, BridgeOffsetNullingCal_ReturnsError)
+{
+  const auto AI_MIN = 0.0;
+  const auto AI_MAX = 1.0;
+  create_ai_voltage_chan(AI_MIN, AI_MAX);
+
+  std::string channel = "";
+  bool skip_unsupported_channels = false;
+  EXPECT_THROW_DRIVER_ERROR({
+    auto response = PerformBridgeOffsetNullingCalExResponse{};
+    auto status = perform_bridge_offset_nulling_cal_ex(channel, skip_unsupported_channels, response);
+  }, BRIDGE_OFFSET_NULLING_CAL_NOT_SUPPORTED_ERROR);
+}
+
+// X Series doesn't support thermocouple offset nulling
+TEST_F(NiDAQmxDriverApiTests, AIThrmcplChannel_ThrmcplLeadOffsetNullingCal_ReturnsError)
```

Comment:
given=AIThrmcplChannel is correct, but given=UnsupportedDevice is clearer.

```suggestion
TEST_F(NiDAQmxDriverApiTests, UnsupportedDevice_ThrmcplLeadOffsetNullingCal_ReturnsError)
```